Habitica/src/main/java/com/habitrpg/android/habitica/ui/activities/PartyInviteActivity.java

```diff
@@ -129,4 +148,42 @@ public CharSequence getPageTitle(int position) {
             tabLayout.setupWithViewPager(viewPager);
         }
     }
+
+    @Override
+    public void onActivityResult(int requestCode, int resultCode, Intent data) {
+        super.onActivityResult(requestCode, resultCode, data);
+
+        IntentResult scanningResult = IntentIntegrator.parseActivityResult(requestCode, resultCode, data);
+
+        if (scanningResult != null) {
+            userIdToInvite = scanningResult.getContents();
+
+            //@TODO: Inject the user
+            this.apiHelper.apiService.getUser()
+                    .compose(this.apiHelper.configureApiCallObserver())
+                    .subscribe(new HabitRPGUserCallback(this), throwable -> {});
+
+        }
+    }
+
+    @Override
+    public void onUserReceived(HabitRPGUser user) {
+        this.user = user;
+
+
+        Toast toast = Toast.makeText(getApplicationContext(),
+                userIdToInvite, Toast.LENGTH_SHORT);
+        toast.show();
+
+        Map<String, Object> inviteData = new HashMap<>();
+        List<String> invites = new ArrayList<>();
+        invites.add(userIdToInvite);
+        inviteData.put("uuids", invites);
+
+        this.apiHelper.apiService.inviteToGroup(this.user.getParty().getId(), inviteData)
```

Comment:
Instead of directly inviting the user, it should just insert the user id into the list of IDs. so that the user then has the option to scan another ID if they want to.